biome.json

```diff
@@ -16,7 +16,13 @@
     }
   },
   "linter": {
-    "enabled": false
+    "enabled": true,
+    "rules": {
+      "recommended": false,
```

Comment:
Disabled recommended rules due to much errors, fixing is out of scope of this PR.

```
The number of diagnostics exceeds the number allowed by Biome.
Diagnostics not shown: 164.
Checked 125 files in 25ms. No fixes applied.
Found 142 errors.
Found 1 warning.
```